internal/cmd/db_replicate.go

```diff
@@ -29,29 +31,77 @@ func replicateArgs(cmd *cobra.Command, args []string, toComplete string) ([]stri
 	return dbNameArg(cmd, args, toComplete)
 }
 
+func pickLocation(locations map[string]string, exclude []string) string {
+	fmt.Printf("Pro-tip! Next time, you can add the %s parameter.\n", internal.Emph("--location"))
+	fmt.Printf("But since we're here... where would you like the replica to be?\n")
+
+	excluded := make(map[string]bool)
+	for _, key := range exclude {
+		excluded[key] = true
+	}
+
+	rows := make([]tbl.Row, 0, len(locations))
+	ids := maps.Keys(locations)
+	sort.Strings(ids)
+
+	for _, id := range ids {
+		if excluded[id] {
+			continue
+		}
+		row := tbl.Row{
+			id,
+			locations[id],
+		}
+		rows = append(rows, row)
+	}
+	columns := []tbl.Column{
+		{Title: "ID↓", Width: 4},
+		{Title: "LOCATION", Width: 32},
+	}
+
+	return prompt.Table(columns, rows, 0)
+
+}
+
 var replicateCmd = &cobra.Command{
 	Use:               "replicate database_name location_id",
 	Short:             "Replicate a database.",
-	Args:              cobra.RangeArgs(2, 3),
+	Args:              cobra.RangeArgs(1, 3),
 	ValidArgsFunction: replicateArgs,
 	RunE: func(cmd *cobra.Command, args []string) error {
+		client, err := createTursoClientFromAccessToken(true)
```

Comment:
This should be after cmd.SilenceUsage = true

Reply:
The problem is that we want the check for args being valid to be before that, no ?

That creates a circularity, since we need the client to show the table.